torch_brain/sampler/distributed_sampler.py

```
@@ -0,0 +1,240 @@
+        """Set the epoch number. Not strictly necessary for sequential sampler
+        but included for API compatibility."""
+        self.epoch = epoch
+
+
+__all__ = [
+    "RandomFixedWindowSampler",
+    "SequentialFixedWindowSampler",
+    "TrialSampler",
+    "DistributedEvaluationSamplerWrapper",
+    "DistributedStitchingFixedWindowSampler",
+]
```

Ruff (0.15.13)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)